client/composer.json

```diff
@@ -25,28 +25,18 @@
         "php-http/guzzle6-adapter": "^2.0",
         "phpunit/phpunit": "^8.0.0",
         "predis/predis": "^1.1.1",
-        "sensio/distribution-bundle": "^5.0.25",
-        "sensio/framework-extra-bundle": "5.4.1",
+        "sensio/framework-extra-bundle": "^6.0",
         "snc/redis-bundle": "^2.1.9",
         "symfony/monolog-bundle": "^3.0.0",
-        "symfony/serializer": "3.4.38",
-        "symfony/symfony": "^3.4",
+        "symfony/symfony": "^4.0",
         "twig/extensions": "^1.5.4"
     },
     "scripts": {
         "post-install-cmd": [
-            "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installAssets",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile"
+            "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters"
         ],
         "post-update-cmd": [
-            "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installAssets",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile"
```

Comment:
Cool if it working but what were these post-install commands actually doing? Does the cache get cleared automatically now post-install?